src/components/BackButton.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+
+const BackButton = () => {
+  const navigate = useNavigate();
+
+  const goBack = () => {
+    navigate("/home");
+  };
+
+  return (
+    <div className="w-8/12 md:w-1/3 m-auto py-3 text-primary dark:text-secondary">
+      {" "}
```

Comment:
use margin or padding instead of this

Reply:
Actually its not required. Should have added while fixing linting issues.
Will remove